Signal/translations/en.lproj/Localizable.strings

```diff
@@ -704,7 +704,7 @@
 "MESSAGE_STATUS_DELIVERED" = "Delivered";
 
 /* message footer for failed messages */
-"MESSAGE_STATUS_FAILED" = "Sending failed. Tap to retry.";
+"MESSAGE_STATUS_FAILED" = "Sending failed. Tap for info.";
```

Comment:
WDYT of this copy change @charlesmchen / @riyapenn?

I didn't want people to hesitate to tap, thinking it would automatically retry. Also might help surface that you can actually get more info by tapping.